var/spack/repos/builtin/packages/gaussian-view/package.py

```diff
@@ -13,29 +14,81 @@ class GaussianView(Package):
     Gaussian. It aids in the creation of Gaussian input files, enables the
     user to run Gaussian calculations from a graphical interface without the
     need for using a command line instruction, and helps in the interpretation
-    of Gaussian output"""
+    of Gaussian output.
+    Needs post-install steps to make it run!
+    See package installation logs for details."""
 
     homepage = "https://gaussian.com/gaussview6/"
     manual_download = True
```

Comment:
Want to be a maintainer for this package too?

Reply:
I'll proudly accept, but should it be me or @dev-zero in this case since he did an awesome job?

Reply:
Why not both? `maintainers` is a list of GitHub handles of people who care about specific packages and want to help review PRs/issues for those packages. The more the better!

Reply:
@dev-zero could you add us both as maintainers?

Reply:
@antoniokaust done, thanks!